SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -203,22 +204,31 @@ Quarks {
 		});
 
 		"Installing %".format(quark.name).postln;
-
+		// add currently installing quark to 'installing' Array
+		installing = (installing.size == 0).if{ quark.bubble }{ installing ++ quark.bubble };
```

Comment:
does appending the quark to `installing` require `.bubble`?

Reply:
Thanks for the review @mtmccrea ! If `installing` is nil then yes you do need `.bubble`: 
```
nil ++Quark("Ctk") //  Quark: Ctk
nil ++ Quark("Ctk").bubble // [Quark: Ctk]
```

Reply:
Ah ok, that makes sense, I was thinking from the perspective of but `installing` always being an array, but it's initial condition is `nil`. But in that case, given the check
```
(installing.size == 0)
``` 
because `nil.size == 0`, `installing` won't ever be `nil` when it hits the false case of `{ installing ++ quark.bubble }`, right?

wrt the use of `bubble`, as I understand `installing` will only ever be a 1D array, and because `bubble` is used for adjusting the shape of a (potentially) multidimensional arrays, perhaps `asArray` would be more direct?

Reply:
I decided to go with `[quark]` - my thinking was if I were reading the code I might imagine that `Quark` has an `asArray` method so might not be immediately sure what `Quark.asArray` returns.

Reply:
Sounds good! Love saving those characters, keep it up and we'll be able to use 'em on something real nice one of these days.